autoload/screen.vim

```diff
@@ -692,7 +692,18 @@ function! screen#IPython(...) " {{{
 
   exec 'ScreenShell' . bang . ' ipython'
 endfunction " }}}
+ 
+function! screen#IPython3(...) " {{{
```

Comment:
Instead of adding a duplicate funtion that differs by 1 line, why not just update the existing function to take an addition argument before the `bang` arg to specify which ipython to run?